Doc/howto/ipaddress.rst

```diff
@@ -255,6 +255,7 @@ membership test syntax like this::
 
    if address in network:
        # do something
+       ...
```

Comment:
I would prefer a single line here and for most of the examples.  Could do the revision with sed.

```suggestion
       ... # do something
```

Reply:
Oops, well spotted, fixed. A follow-up PR could reformat *all* the examples consistently; this PR only works on the places where there were parsing problems.